lighthouse-core/scripts/lantern/download-traces.sh

```diff
@@ -13,7 +13,7 @@ cd $LH_ROOT_PATH
 if [[ -f lantern-data/version ]] && [[ "$VERSION" != "$(cat lantern-data/version)" ]]; then
   if ! [[ "$CI" ]]; then
     echo "Version out of date. About to delete ./lantern-data..."
-    echo "Press any key to continue, Ctrl+C to exit"
```

Comment:
it should be pretty much any key that prints a character that makes this work, just want something specific that definitely works?

Reply:
Arrow keys exit the script :/

Reply:
good to know 👍